packages/devtools_app/test/test_infra/flutter_test_driver.dart

```diff
@@ -48,13 +48,6 @@ abstract class FlutterTestDriver {
 
   VmServiceWrapper? vmService;
 
-  String get lastErrorInfo => errorBuffer.toString();
```

Comment:
These could be helpful for debugging in the future. We could probably leave them.

Reply:
Thanks, done!